e2e/tests/schema.spec.ts

```
@@ -0,0 +1,337 @@
+        await apicalls.removeSchema(schemaName);
+    });
+
+    test(`@admin validate that deleting a schema node decreases node count`, async () => {
+        const schemaName = getRandomString('schema');
+        await apicalls.addSchemaNode(schemaName, 'CREATE (a:person1 {id: "Integer!*-1"}), (b:person2 {id: "Integer!*-2"}) RETURN a, b');
+        await waitForApiSuccess(() => apicalls.getSchemas(), res => res.opts.includes(schemaName));
+        const schema = await browser.createNewPage(SchemaPage, urls.schemaUrl);
+        await browser.setPageToFullScreen();
+        await schema.selectExistingGraph(schemaName);
+        const nodes = await schema.getNodeScreenPositions('schema');
+        const initCount = parseInt(await schema.getNodesGraphStats() ?? "");
```

ESLint

[error] Missing radix parameter.

(radix)